### ISSUE TITLE
Strange rendering effects

### ISSUE THREAD

**Issue by bitnapper** [osmandapp/Osmand#2374]
![Sankt Andreasberg, Germany](https://pbs.twimg.com/media/CelCC7tWAAAUdyE.jpg:large)

I have some strange rendering effects. I couldn't find anything wrong in the osm data so it might be an issue with the rendering but I can't tell for shure. Maybe its related to Issue #2370. The images shows the vicinity of [Sankt Andreasberg](http://osm.org/go/0GtqbI8A-), Germany. I removed the files from OSM Live but that had no effect.

**Comment by vshcherb** [comment]
Yes it looks very bad but I remember that issue only with OSM Live, I had to remove all files in osmand/live folder. It also could be related to SRTM as I see but it was never a case before.

**Comment by RalfJung** [comment]
Seems like <https://github.com/osmandapp/Osmand/issues/2370> has been closed as a duplicate of this?

**Comment by vshcherb** [comment]
#2370 has nothing to do with #2374 as I think there is another issue with these lines and live udpate.

**Comment by RalfJung** [comment]
But then why was #2370 closed?

**Comment by CJTmmr** [comment]
On Osmand 2.3.3 (Play Store) and latest nightly's:

I have installed maps from Italy Piemonte, Switzerland, Germany (all Western Germany-maps), The Netherlands, Belgium.

With some Live-Update-maps present (only for Netherlands en Belgium) in map Live.
Same strange straight lines visible in zoom 13-20 (not visible in zoom 1-12)
Directiong between NNE and N.
Even with LiveUpdate not enabled in Side Menu.....

All the way from Punta Garavano = http://www.openstreetmap.org/node/1279765785#map=16/43.7835/7.5403
(which is outside of my maps..in Italy Liguria.)
![2016-03-20 14 37 57](https://cloud.githubusercontent.com/assets/15076257/14123427/c052300c-f600-11e5-977d-38bea2221f55.png)

through Switzerland

to Germany near Oberzissen: http://www.openstreetmap.org/relation/1050577#map=14/50.4519/7.2035
![2016-03-19 13 25 18](https://cloud.githubusercontent.com/assets/15076257/14124311/1e51e798-f605-11e5-9f79-cda24c5b9fca.png)



The lines also mesh up rendering of rivers etc. flooding large areas.
I did not check where the lines ends in the NorthNorthEast....

If I delete all Live Maps (or rename the Live-map) and after restart of OsmAnd (Force Stop/Clear Cache with application-manager) the straight lines no longer appear.

BTW Osmand+ from PlayStore and OsmandF~ from nightly can be installed next to each other again - great job!
